lang/en/docs/jupyterlite/data-exchange.md

```diff
@@ -0,0 +1,39 @@
+# Data Exchange with JupyterLite
+
+## Overview
+
+JupyterLite environment can exchange data (1) either directly with the platform or (2) with its sub-parts, such as the [Materials Designer](../materials-designer/overview.md).
+
+## Get data inside JupyterLite
+
+`get_data` is used to init a local variable and make it available inside the session.
+
+For example, to work with materials data inside the JupyterLite environment inside [Materials Designer](../materials-designer/overview.md), use the following code snippet:
+
+```python
+from utils.jupyterlite import get_data
+
+get_data("materials_in", globals())
+```
+
+The first parameter specifies the name of the global variable (`"materials_in"`) where the received data will be stored. The second parameter, `globals()`, ensures the function operates correctly across both Pyodide and Python environments. It allows `get_data` to dynamically interact with the global namespace of the script.
```

Comment:
Pyodide needs to be explained somewhere with links to it

Reply:
Should have a separate page for pyodide and explain the basics and how to install packages and do basic operations like python.ipynb from examples